tests/nemo_text_processing/de/data_text_normalization/test_cases_electronic.txt

```diff
@@ -6,4 +6,11 @@ a eins b zwei at a b c punkt com~a1b2@abc.com
 a b drei bindestrich s d d bindestrich drei at g mail punkt com~ab3-sdd-3@gmail.com
 h t t p s doppelpunkt slash slash w w w punkt a b c punkt com~https://www.abc.com
 w w w punkt a b c punkt com~www.abc.com
-h t t p s doppelpunkt slash slash w w w punkt a b c punkt com slash a b fragezeichen gleichheitszeichen drei bindestrich slash a b s slash eins~https://www.abc.com/ab?=3-/abs/1
\ No newline at end of file
+b r e t t s p i e l v e r s a n d punkt de .~brettspielversand.de.
+w w w punkt z u b e r e k punkt net .~www.zuberek.net.
+w w w punkt a m a z o n punkt com punkt de .~www.amazon.com.de.
+h t t p s doppelpunkt slash slash w w w punkt a b c punkt com slash a b fragezeichen gleichheitszeichen drei bindestrich slash a b s slash eins~https://www.abc.com/ab?=3-/abs/1
+at z u c k~@zuck
+at z o o b e r e q~@zoobereq
```

Comment:
Don't use your name as an example. It leads to potential doxing.

Reply:
Good point - thank you.  Will fix.